### ISSUE TITLE
conda cannot install same version of itself from a file:/// channel on Windows 

### ISSUE THREAD

**Issue by jjhelmus** [conda/conda#2876]
On Windows, performing `conda install -c file:///path_to_channel package_that_depend_on_conda` which triggers an install of the same version of conda but from the file:/// channel (a SUPERCEDED update) fails with the follow traceback:

```
conda install --yes --quiet conda-build-all
......
Solving package specifications: ..........
Package plan for installation in environment C:\Miniconda:
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    conda-env-2.5.1            |           py27_0          63 KB  file:///c:/package-cache
    conda-4.1.3                |           py27_0         240 KB  file:///c:/package-cache
    conda-build-1.21.2         |           py27_0         307 KB  file:///c:/package-cache
    conda-build-all-0.13.0     |           py27_1          86 KB  file:///c:/package-cache
    ------------------------------------------------------------
                                           Total:         697 KB
The following NEW packages will be INSTALLED:
    conda-build-all: 0.13.0-py27_1 file:///c:/package-cache
The following packages will be UPDATED:
    conda-build:     1.20.1-py27_0 defaults                 --> 1.21.2-py27_0 file:///c:/package-cache
The following packages will be SUPERCEDED by a higher-priority channel:
    conda:           4.1.3-py27_0  defaults                 --> 4.1.3-py27_0  file:///c:/package-cache
    conda-env:       2.5.1-py27_0  defaults                 --> 2.5.1-py27_0  file:///c:/package-cache
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:
    https://github.com/conda/conda/issues
Include the output of the command 'conda info' in your report.
Traceback (most recent call last):
  File "C:\Miniconda\Scripts\conda-script.py", line 5, in <module>
    sys.exit(main())
  File "C:\Miniconda\lib\site-packages\conda\cli\main.py", line 120, in main
    exit_code = args_func(args, p)
  File "C:\Miniconda\lib\site-packages\conda\cli\main.py", line 130, in args_func
    exit_code = args.func(args, p)
  File "C:\Miniconda\lib\site-packages\conda\cli\main_install.py", line 69, in execute
    install(args, parser, 'install')
  File "C:\Miniconda\lib\site-packages\conda\cli\install.py", line 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Miniconda\lib\site-packages\conda\plan.py", line 591, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Miniconda\lib\site-packages\conda\instructions.py", line 136, in execute_instructions
    cmd(state, arg)
  File "C:\Miniconda\lib\site-packages\conda\instructions.py", line 59, in EXTRACT_CMD
    extract(arg)
  File "C:\Miniconda\lib\site-packages\conda\install.py", line 847, in extract
    exp_backoff_fn(os.rename, temp_path, path)
  File "C:\Miniconda\lib\site-packages\conda\install.py", line 235, in exp_backoff_fn
    result = fn(*args)
WindowsError: [Error 183] Cannot create a file when that file already exists
Command exited with code 1
```

**Comment by mcg1969** [comment]
I think this may be related to the open-file deletion problem we discovered and fixed Friday. Hang on.

**Comment by mcg1969** [comment]
Great, I am indeed able to reproduce this. @kalefranz I'll diagnose.

**Comment by kalefranz** [comment]
fixed in #2882

**Comment by jjhelmus** [comment]
Thanks for the quick fix guys!
